### PR TITLE
chore : Migrate Robocon section to firestore

### PR DESCRIPTION
**Pull request by Apoorve73**
# Overview

Update Queries for Robocon page, to fetch data from Cloud Firestore instead of Real Time Database.

### CODE REVIEW

File: plugins/firebase.js [hunk 1]
```diff
@@ -13,6 +14,7 @@ if (!firebase.apps.length) {
 }
 
 export const db = firebase.database()
+export const firestoreDb = firebase.firestore()
```
Comment (HashirGJ8842) [inline plugins/firebase.js]: Rename this variable as `DB`, this would be the name everyone will proceed with

**Review by HashirGJ8842 — COMMENTED** [review]
Nice work

**Comment by HashirGJ8842** [comment]
And update this branch to resolve conflicts.

File: plugins/firebase.js (hunk 1 above)
Comment (Apoorve73) [reply plugins/firebase.js]: Sure!

**Comment by Apoorve73** [comment]
I have resolved conflicts and have also made the required changes. Please check now, @HashirGJ8842 
Thanks!

**Review by HashirGJ8842 — APPROVED** [review]
LGTM!

**Pull request merged**